qml/components-Sailfish/IconButton.qml

```diff
@@ -0,0 +1,30 @@
+import Sailfish.Silica 1.0
+import "." as BVApp
+
+IconButton {
+    function iconBy(type) {
```

Comment:
That ```IconButton``` abstraction is amazing, nice! I have also already thought about this a bit (when thinking about the Theme), and I head exactly something like this in mind ;). Of course, I haven't implemented it / tried it out. Very nice work!

One idea: What would you think about moving this ```iconBy()``` function to the Theme class? Maybe we could do sth. like 
```
Theme
{
...
  function icon(type, scale)
...
}
```

I see the following advantages with that approach:
1. We don't have provide a wrapper for the ```IconButton``` implementation.
2. We can use this class in other cases where we need icons with native look and feel.